### ISSUE TITLE
Allow participants to get credit for recruiting new participants - SCOPING

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#622]
# TL;DR
Set up to collect a "referral code" field for new participants and view codes of new participants.

# Narrative
Participant word-of-mouth referrals are valuable and could be substantially more efficient than paid advertising if we could compensate families directly for referring others.

# TODO during scoping

- Confirm with MIT whether we can compensate families for referrals
- Confirm with MIT what options we have on compensation (non-Amazon) if so

# Acceptance Criteria - TODO, DRAFT

- [ ] Participants can see a referral link or code in their account view, with instructions about how it works
- [ ] Someone who follows a referral link or enters a referral code upon registration has that code saved with their account
- [ ] A staff member with a specific permission can see a list of participants including ID, referral code, participant code the referral belongs to if appropriate, and an editable "processed" field for each of the two participants (e.g. to cover whether they have been compensated for the referral)
- [ ] Shortcut buttons allow emailing participants from the above view

# Implementation Notes
